python-backend/app/api/permits/permit_amendment/models/permit_amendment.py

```diff
@@ -24,14 +24,16 @@ class PermitAmendment(AuditMixin, Base):
     permit_amendment_type_code = db.Column(
         db.String(3), db.ForeignKey('permit_amendment_type_code.permit_amendment_type_code'))
 
+    deleted_ind = db.Column(db.Boolean, nullable=False, server_default=FetchedValue())
+
     def json(self):
         return {
             'permit_amendment_id':
-            self.permit_amendment_id,
+            str(self.permit_amendment_id),
             'permit_amendment_guid':
             str(self.permit_amendment_guid),
-            'permit_id':
-            self.permit_id,
+            'permit_guid':
+            str(self.permit.permit_guid),
```

Comment:
What happens if you don't convert a guid to a string here? This seems redundant to me

Reply:
objects of type uuid aren't serializable by default sadly.

Reply:
Okay! Weird